Tasks/BuildctlV0/Strings/resources.resjson/fr-fr/resources.resjson

```diff
@@ -0,0 +1,10 @@
+{
+  "loc.friendlyName": "Programme d'installation de l'outil Kubectl",
+  "loc.helpMarkDown": "[Plus d'informations](https://go.microsoft.com/fwlink/?linkid=851275)",
+  "loc.description": "Installer Kubectl sur une machine d'agent",
+  "loc.instanceNameFormat": "Installer Kubectl $(kubectlVersion)",
+  "loc.input.label.kubectlVersion": "Spécification de version de Kubectl",
+  "loc.input.help.kubectlVersion": "Spécifier la version de Kubectl à installer",
+  "loc.messages.NotAValidSemverVersion": "Version non spécifiée au format approprié. Exemple : 1.8.2, v1.8.2, 2.8.2, v2.8.2.",
```

Comment:
I don't think you need to check in the loc files, at all. Can you please remove the non enu files. The loc team should then go and convert the strings to the respective languages.

Reply:
removed the extra loc files